Mathlib/LinearAlgebra/Alternating.lean

```diff
@@ -115,24 +115,29 @@ theorem toFun_eq_coe : f.toFun = f :=
   rfl
 #align alternating_map.to_fun_eq_coe AlternatingMap.toFun_eq_coe
 
+-- Porting note: New theorem.
 @[simp]
-theorem coe_mk (f : (ι → M) → N) (h₁ h₂ h₃) : ⇑(⟨f, h₁, h₂, h₃⟩ : AlternatingMap R M N ι) = f :=
+theorem coe_mk (f : MultilinearMap R (fun _ : ι => M) N) (h) :
+    ⇑(⟨f, h⟩ : AlternatingMap R M N ι) = f :=
+  rfl
+
+theorem coe_mks (f : (ι → M) → N) (h₁ h₂ h₃) : ⇑(⟨⟨f, h₁, h₂⟩, h₃⟩ : AlternatingMap R M N ι) = f :=
```

Comment:
I think the thing to do would be to delete this lemma, and align `alternating_map.coe_mk` with `alternating_map.coe_mk\_x`

Reply:
I've just pushed a better fix for this and `coe_multilinearMap_mk`. Noting that the mathport output has
```lean
theorem coe_mk (f : (ι → M) → N) (h₁ h₂ h₃) : ⇑(⟨f, h₁, h₂, h₃⟩ : AlternatingMap R M N ι) = f :=
  rfl

theorem coe_multilinearMap_mk (f : (ι → M) → N) (h₁ h₂ h₃) :
    ((⟨f, h₁, h₂, h₃⟩ : AlternatingMap R M N ι) : MultilinearMap R (fun i : ι => M) N) =
      ⟨f, @h₁, @h₂⟩ :=
  rfl
```
and that these don't work, but putting angle brackets in the right places (as already existed in this PR) fixes everything, it only makes sense to just add the brackets and leave porting notes explaining the new `mk` structure.